static/docs/commands-reference/remote_remove.md

```diff
@@ -36,8 +36,8 @@ possible to edit config files manually.
 
 - `--local` - remove remote specified in the
   [local](/doc/user-guide/dvc-files-and-directories) configuration file
-  (`.dvc/config.local`). Local configuration files stores private settings or
-  local environment specific settings that should not be tracked by Git.
+  (`.dvc/config.local`). Local config files stores private configuration that
```

Comment:
stores -> store

Reply:
Fixed in ddece21

Reply:
Actually... For some reason this option had a different working in `remote list` and `remote remove`. I standardized it with the rest of the `remote` subcommands in a69f0ae.